app/src/lib/window-state.ts

```diff
@@ -56,7 +56,7 @@ export function registerWindowStateChangedEvents(
  * Short hand convenience function for sending a window state change event
  * over the window-state-changed channel to the render process.
  */
-function sendWindowStateEvent(
+export function sendWindowStateEvent(
```

Comment:
Did we need this export?

```suggestion
function sendWindowStateEvent(
```